src/migrations/00014-modified-at.sql

```diff
@@ -0,0 +1,84 @@
+-- The commented sections of this file must be manually run as they affect models
```

Comment:
This is the biggest question around the change, since there are some built in tables that have to be manually migrated due to being required for the migrations system to actually get up and running